etc/docker/Dockerfile.triton-jetpack-focal

```diff
@@ -1,154 +1,147 @@
 FROM ubuntu:focal AS base
-
 ARG DEBIAN_FRONTEND=noninteractive
 
-RUN apt-get -y update
-RUN apt-get -y dist-upgrade
-
 # Add components needed for apt-key / apt-add-repository, since
 # we need to have the JetPack repo available for both build and runtime
 #
 # Also sudo, since both jetpack installation postinsts and the runtime
 # need it.
-RUN apt-get -y --no-install-recommends install \
+RUN apt-get -y update && apt-get -y dist-upgrade && apt-get -y --no-install-recommends install \
     ca-certificates \
     gnupg \
     sudo
 
 RUN apt-key adv --fetch-key http://repo.download.nvidia.com/jetson/jetson-ota-public.asc
-RUN echo 'deb https://repo.download.nvidia.com/jetson/common r35.4 main' > /etc/apt/sources.list.d/nvidia-jetson.list
-RUN apt-get -y update
+RUN echo 'deb https://repo.download.nvidia.com/jetson/common r35.5 main' > /etc/apt/sources.list.d/nvidia-jetson.list
 
 
 FROM base AS runtime
-
 ARG DEBIAN_FRONTEND=noninteractive
 
-# Runtime for gRPC and friends
-RUN apt-get -y --no-install-recommends install \
-    libc-ares2 \
-    libre2-5 \
-    libssl1.1 \
-    zlib1g
-
 # Hack to get PVA stuff to install
 RUN mkdir -p /lib/firmware
 
-# Runtime components of JetPack that we need
-RUN apt-get -y --no-install-recommends install \
+# Install packages for runtime for:
```

Comment:
This regrouping to a single apt-get was intended to improve caching. I don't know if it worked or not.